game.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body onmousedown="showCoords(event)">
+    <canvas id="main" width="360" height="360"></canvas>
```

Comment:
use class instead of id

Reply:
I don't know class in js

Reply:
<canvas class="main" width="360" height="360"></canvas>
<script>
var el = document.getElementsByClass('main')[0];
</>